my-express-app/server.js

```diff
@@ -50,6 +50,29 @@ app.get("/videos", async (req, res) => {
     res.status(500).json({ message: err.message });
   }
 });
+app.get("/recommendations", async (req, res) => {
+  const selectedDifficulty = req.query.Difficulty;
+  const selectedAgeGroup = req.query.AgeGroup;
+  const selectedInterests = req.query.Interests;
+
+  try {
+    // Use Sequelize ORM to find the matching videos in the database
+    const videos = await Videos.findAll({
```

Comment:
I am unsure why this is in this file. Will this throw an error if the database does not find a video that does not match all the parameters? Surely this is bound to happen.

I know we discussed offline tweaking the algorithm so I am not sure if this gets changed in later pull requests